### PR TITLE
Analyses List Refactor to Ant Design

### LINKED ISSUE #143: Add a method to batch-delete analysis pipelines
Imported from GitLab issue #613.  Originally posted on 2018/02/16 03:20PM

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Update both project and global analyses listing pages to use Ant Design tables.

## Related issue
#413 
Fixes #143 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [X] Tests added (or description of how to test) for any new features.
* [X] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/webapp/resources/js/pages/projects/list/ProjectsTable.jsx [hunk 1]
```diff
@@ -186,35 +187,34 @@ export function ProjectsTable() {
...
-        pagination={{
-          total: total,
-          pageSize: state.pageSize
-        }}
-        columns={columns}
-        dataSource={projects}
-        onChange={handleTableChange}
-      />
+    <PageWrapper title={getI18N("ProjectsTable_header")}>
+      <>
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/list/ProjectsTable.jsx]: These empty brackets aren't needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Changes
 19.09 to 20.01
 --------------
 * [Developer]: Upgraded development Java version to Java 11 (LTS).
+* [UI]: Added the ability to delete multiple analyses from both the Project > Analyses and List All Analyses pages.
+* [UI]: Updated project and global analyses listing pages to use Ant Design tables.
```
Comment (deepsidhu85) [inline CHANGELOG.md]: The Sequencing Runs table was also updated so we should add it in here

File: src/main/webapp/resources/js/pages/projects/list/ProjectsTable.jsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/list/ProjectsTable.jsx]: Removed in c3667bba615bc258475e35681e1f68f6f7a210fb

File: CHANGELOG.md (hunk 2 above)
Comment (joshsadam) [reply CHANGELOG.md]: Updated in 5dfe3c3ed5baf6fdc962abff1f3cc164f557c73a

**Pull request merged**